### PR TITLE
feat: create tooltip component

### PR DESCRIPTION
**Pull request by di-kinosian**

### CODE REVIEW

File: src/components/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Popover } from '../Popover/Popover';
+
+export interface TooltipProps {
+  text: string;
+  position?: 'top' | 'right' | 'bottom' | 'left';
```
Comment (makhynenko) [inline src/components/Tooltip/Tooltip.tsx:6]: move to global types as PopoverPositionType = 'top' | 'right' | 'bottom' | 'left' and use here :
position: PopoverPositionType

File: src/components/Tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  align?: 'start' | 'center' | 'end';
```
Comment (makhynenko) [inline src/components/Tooltip/Tooltip.tsx:7]: Same as above

File: src/components/Tooltip/Tooltip.tsx (hunk 1 above)
Comment (di-kinosian) [reply src/components/Tooltip/Tooltip.tsx:6]: @makhynenko done

File: src/components/Tooltip/Tooltip.tsx (hunk 2 above)
Comment (di-kinosian) [reply src/components/Tooltip/Tooltip.tsx:7]: @makhynenko done

**Review by makhynenko — APPROVED** [review]

**Pull request merged**